AY2021S2-CS2103-T14-1/tp
# Fix trailing duplicate prefixes issue

## What this does
This PR
1. fixes the issues where the system would parse the second prefix instead of the first one (e.g. `alias a/h cmd/help cmd/clear` would map `h` to `clear` instead of `help cmd/clear` as an expected behavior.

## How to test
1. Launch SunRez.
2. Execute the `alias a/h cmd/help cmd/clear` command with the name of the previously created alias. 
3. Execute the alias. You should see a command not found error.
4. Check the preference json file to see that `h` is mapped to `help cmd/clear`.
